test/features/support_archive/support_archive.go

```diff
@@ -209,3 +218,42 @@ func logMissingFiles(t *testing.T, requiredFiles []string) {
 		t.Log(missingFilesLog)
 	}
 }
+
+func logMessage(dk dynakube.DynaKube, podName string, containerName string, message string) features.Func {
+	return func(ctx context.Context, t *testing.T, envConfig *envconf.Config) context.Context {
+		resource := envConfig.Client().Resources()
+		clientset, err := kubernetes.NewForConfig(resource.GetConfig())
+		require.NoError(t, err)
+
+		err = wait.For(func(ctx context.Context) (done bool, err error) {
+
+			logStream, err := clientset.CoreV1().Pods(dk.Namespace).GetLogs(podName, &corev1.PodLogOptions{
+				Container: containerName,
+			}).Stream(ctx)
+			if err != nil {
+				return false, err
+			}
+
+			buffer := new(bytes.Buffer)
+			copied, err := io.Copy(buffer, logStream)
+			if err != nil {
+				return false, err
+			}
+			if int64(buffer.Len()) != copied {
+				return false, fmt.Errorf("incorrect number of bytes copied (%d vs %d)", buffer.Len(), copied)
+			}
+
+			if strings.Contains(buffer.String(), message) {
+				t.Log("found")
```

Comment:
if you want to keep this log, it's better to add message as well: "found %s", message